ui/src/components/common/CopyToClipboardMultiline.js

```diff
@@ -1,36 +1,35 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { OverlayTrigger, Tooltip } from 'patternfly-react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import classNames from 'classnames';
 
 import './CopyToClipboardMultiline.css';
 
-class CopyToClipboardMultiline extends Component {
-  render() {
-    return (
-      <div
-        className={classNames('input-group', 'copy-to-clipboard-multiline', this.props.className)}
-      >
-        <pre>
-          {this.props.children}
-        </pre>
-        <span>
-          <CopyToClipboard text={this.props.children}>
-            <OverlayTrigger
-              overlay={<Tooltip id="tooltip">Copied!</Tooltip>}
-              placement="left"
-              trigger={['click']}
-              rootClose
-            >
-              <a className="btn btn-default copy-btn">
-                Copy
-              </a>
-            </OverlayTrigger>
-          </CopyToClipboard>
-        </span>
-      </div>
-    );
-  }
-}
+const spanizeString = code =>
+  code.split('\n').reduce(
+    (accu, line) => (
+      <React.Fragment>
+        {accu}
+        <span>{line}</span>
+      </React.Fragment>
```

Comment:
is it necessary to return the outer `React.Fragment` here?

Reply:
I actually can't get MDC to work with this settings.

Reply:
It's how the React works with the DOM, I think. I don't know if there is other option. I'll check it with @jhellar.

Reply:
@wei-lee @wojta works for me. IMO the problem is that jsx won't allow for more than one root tag in one expression - removing `<React.Fragment>` here would not work. But there might be another problem - reduce in this way with every line will wrap the previous one in another fragment. IMO it should not be a problem even for many lines. But there is other possible solution, which won't do the wrapping:
```
const spanizeString = code => (
  <React.Fragment>
    {code.split('\n').map((line, index) => (
      <span key={index}>{line}</span>
    ))}
  </React.Fragment>
);
```

Reply:
thanks @jhellar , I used your code.